Dockerfile.test

```diff
@@ -1,3 +1,3 @@
-FROM davidtnfsh/pycon_etl:cache
+FROM davidtnfsh/pycon_etl:prod
 ENV AIRFLOW_TEST_MODE=True
 USER root
```

Comment:
ditto

Reply:
haha sure. Btw, would u recommend some Dockerfile linter for this kind of nitpicks?

Reply:
FYI: most mainstream IDE or editor should support "no end of line" automatic fix. PyCharm, VIM ...etc.